tests/scalability/scale-test.sh

```diff
@@ -2,21 +2,21 @@
 
 source ./util.sh
 
-GSID=85ffe8da-c82f-4035-86c5-9d2b5f42d6f7
+GSB_NAME=gameserverbuild-sample-openarena
 
 echo "test 1: scale up to 16 servers from 1 standby server"
 kubectl apply -f ./standby/1.yaml
-scale_up $GSID 16
+scale_up $GSB_NAME 16
 kubectl delete gsb gameserverbuild-sample-openarena
```

Comment:
do we need to delete the build before scaling? we can downscale to zero if needed

Reply:
We can scale down to 0. Which one is more appropriate here?

Reply:
let's scale to zero, any chance we can wait till the actual standing by is zero?

Reply:
Done